server/hotrod/src/main/scala/org/infinispan/server/hotrod/iteration/IterationManager.scala

```diff
@@ -17,6 +17,7 @@ import org.infinispan.server.hotrod.logging.Log
 
 import scala.collection.JavaConversions._
 import scala.collection.mutable
+import scala.language.existentials
```

Comment:
not necessary, but it says that existential type is used intentionally here and thus removes the feature warning during compilation.